src/components/common/ValidationFunctions.res

```diff
@@ -27,3 +27,17 @@ let isValidZip = (~zipCode, ~country) => {
   let isZipCodeValid = RegExp.test(postalCode.regex->Js.Re.fromString, zipCode)
   zipCode->String.length > 0 && isZipCodeValid
 }
+
```

Comment:
Not sure of keeping these functions in this file or `Validation.res`. Please help!

Reply:
Moved functions from `ValidationFunction.res` to `Validation.res` #148